src/bus/lattice.rs

```diff
@@ -77,6 +96,92 @@ impl DistributedBus {
     }
 }
 
+fn spawn_inventory_handler(
+    nc: Arc<nats::Connection>,
+    host_id: String,
+    claims: Arc<RwLock<HashMap<String, Claims<Actor>>>>,
+    bindings: Arc<RwLock<BindingsList>>,
+    caps: Arc<RwLock<HashMap<RouteKey, CapabilityDescriptor>>>,
+) -> Result<()> {
+    let _ = nc.subscribe(INVENTORY_SUBJECT)?.with_handler(move |msg| {
```

Comment:
Is there a reason you're assigning the result of this `subscribe` method to nothing?

Reply:
Yep, this is a common pattern for signaling that we intend to invoke a result-returning function, but we don't care if it fails. There is a larger concern outside the scope of this PR where we need to address some of the patterns we're following for error handling in background threads.